documentation/topics/glossary.md

```diff
@@ -0,0 +1,95 @@
+# Glossary
+
+(This is a work in progress and will be expanded over time!)
+
+## Action
+
+An action describes an operation that can be performed for a given resource; it is the _verb_ to a resource's _noun_. Examples of actions:
+
+- User._create_
+- Comment._delete_
+- BlogPost._publish_
+- Article._search_
+
+Ash supports four different types of actions - `create`, `read`, `update` and `destroy` (collectively often abbreviated as CRUD). A resource can define multiple actions per action type, eg. a _publish_ action would be considered an `update` because it is updating an existing instance of a resource.
+
+See the [Actions guide](/documentation/topics/actions.md) for more information.
+
+## Actor
+
+The entity that performs an action. Most actions are run on direct user request, eg. if a user presses a Create button on a page then the actor is the user; but an actor might also be an organization, a group, or the system itself.
+
+Actors are referenced during authorization - ensuring that the actor is allowed to perform an action, before it takes place.
+
+See the [Security guide](/documentation/topics/security.md#actors) for more information.
+
+## Aggregate
+
+An aggregate is a special type of attribute for a resource, one that summarizes related information of the record. A more specialized type of a [calculation](#calculation).
+
+If a Project resource has_many Ticket resources, an example of an aggregate on the Project might be to count the tickets associated to each project.
+
+See the [Aggregates guide](/documentation/topics/aggregates.md) for more information.
+
+## API
+
+## Attribute
+
+A piece of data belonging to a resource. The most basic building block; an attribute has a type and a value amd is stored within the context of a domain model.
+
+See `d:Ash.Resource.Dsl.attributes` for more information.
+
+## Calculation
```

Comment:
In Ash terms we'd call a `calculation` a "field", as opposed to a special attribute, generally speaking. There are four types of fields: attributes, calculations, aggregates, and relationships. I also might suggest flipping around the bit about aggregates, and putting it on the aggregate definition (which I see you've already put on there, so maybe just remove it from here? I'm on the fence so its your call).